### PR TITLE
Create QR code card - Overlapping views issue (EXPOSUREAPP-6544)

### PR DESCRIPTION
**Pull request by BMItr**
This PR addresses Exposureapp-6544. (TraceLocationCard - Overlapping views on small font sizes etc.)

How to Test:
- change the font size in your os settings to small, medium, big
- open the CWA App (MainFragment)
- Check the "Planing an Event Card" for overlapping issues

Screenshot (before):
image

Screenshot (now, small & normal fontsize):
now_normalsize
eventcard_nowsize_small

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/home_create_trace_location_card.xml [hunk 1]
```diff
@@ -30,26 +28,36 @@
         android:text="@string/create_trace_location_card_subtitle"
         app:layout_constraintEnd_toStartOf="@id/create_trace_location_card_icon"
         app:layout_constraintStart_toStartOf="@id/create_trace_location_card_headline"
-        app:layout_constraintTop_toBottomOf="@id/create_trace_location_card_headline" />
+        app:layout_constraintTop_toBottomOf="@id/create_trace_location_card_headline"
+        tools:text="@string/create_trace_location_card_subtitle" />
 
     <ImageView
```
Comment (AlexanderAlferov) [inline Corona-Warn-App/src/main/res/layout/home_create_trace_location_card.xml:34]: Afaik we are trying to streamline every image view to `<androidx.appcompat.widget.AppCompatImageView`
Would be great to for this small refactoring in your ticket 🤖

**Review by AlexanderAlferov — DISMISSED** [review]
Looks great:
screenshot-1619095001902
Only one small comment 🚀

File: Corona-Warn-App/src/main/res/layout/home_create_trace_location_card.xml (hunk 1 above)
Comment (BMItter) [reply Corona-Warn-App/src/main/res/layout/home_create_trace_location_card.xml]: @AlexanderAlferov 
thx for the info. adjusted

**Review by AlexanderAlferov — APPROVED** [review]

**Review by chiljamgossow — APPROVED** [review]

File: Corona-Warn-App/src/main/res/layout/home_create_trace_location_card.xml (hunk 1 above)
Comment (fynngodau) [reply Corona-Warn-App/src/main/res/layout/home_create_trace_location_card.xml]: @AlexanderAlferov As I understood in https://github.com/corona-warn-app/cwa-app-android/pull/2700#pullrequestreview-622693686, writing `ImageView` is the same as writing `androidx.appcompat.widget.AppCompatImageView`. The only thing that needs to be looked out for is using `android.widget.ImageView`, as that is not the same and won't make use of the compat libraries.

**Pull request merged**